modules/manually-removing-cloud-creds.adoc

```diff
@@ -21,5 +22,15 @@ Prior to a non z-stream upgrade, you must reinstate the credential secret with t
 . In the *Administrator* perspective of the web console, navigate to *Workloads* -> *Secrets*.
```

Comment:
For consistency, if you change the intro text for 4.5, please apply it here, too.